apps/prairielearn/package.json

```diff
@@ -247,7 +247,7 @@
     "@types/tmp": "^0.2.6",
     "@types/unzipper": "^0.10.11",
     "@types/uuid": "^10.0.0",
-    "axe-core": "^4.10.2",
+    "axe-core": "^4.10.3",
```

Comment:
We can't consume this because of https://github.com/salesforce/sa11y/issues/586.